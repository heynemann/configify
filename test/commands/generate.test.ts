import { stdout } from 'stdout-stderr'
import Generate from '../../src/commands/generate'

test('generate command with force', async () => {
  stdout.start()
  const cmd = new Generate(['-f', './test/config'], null)
  await cmd.run()
  stdout.stop()
  expect(stdout.output).toMatchSnapshot()
})
