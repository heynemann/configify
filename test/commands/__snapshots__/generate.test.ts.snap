// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate command with force 1`] = `
"ⓘ environment successfully read from /Users/heynemann/dev/vtex/configify/test/config/env.ts.

Configuration updated successfully for paths:
⟳ /Users/heynemann/dev/vtex/configify/test/config/vtex/config.ts
⟳ /Users/heynemann/dev/vtex/configify/test/config/vtex/customer1/config.ts
⟳ /Users/heynemann/dev/vtex/configify/test/config/vtex/customer1/master/config.ts
⟳ /Users/heynemann/dev/vtex/configify/test/config/vtex/customer1/master/workspace1/config.ts
⟳ /Users/heynemann/dev/vtex/configify/test/config/vtex/customer1/master/workspace2/config.ts
"
`;
